independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/handlers/ResourceLocatorHandler.java

```diff
@@ -109,8 +111,9 @@ public void onComplete(Throwable throwable) {
         if (mapper == null) {
             throw new WebApplicationException(Response.status(Response.Status.METHOD_NOT_ALLOWED.getStatusCode()).build());
         }
-        RequestMapper.RequestMatch<RuntimeResource> res = mapper
-                .map(requestContext.getRemaining().isEmpty() ? "/" : requestContext.getRemaining());
+        if (res == null) {
+            res = findRequestMatch(mapper, requestContext);
```

Comment:
This again looks like we could invoke `findRequestMatch()` twice with the same parameters, if it returned `null` the first time.